src/sentry/search/events/datasets/discover.py

```diff
@@ -1578,6 +1594,104 @@ def _resolve_percentile(
             )
         )
 
+    def _resolve_web_vital_score_function(
+        self,
+        args: Mapping[str, Column],
+        alias: str,
+    ) -> SelectType:
+        column = args["column"]
+        if column.key not in [
+            "score.lcp",
+            "score.fcp",
+            "score.fid",
+            "score.cls",
+            "score.ttfb",
+        ]:
+            raise InvalidSearchQuery(
```

Comment:
I would add a test to verify that this gets raised when the column.key is not in the list